src/applications/vaos/components/calendar/CalendarWidget.jsx

```diff
@@ -46,10 +74,16 @@ export function getMaxMonth(maxDate, startMonth) {
   return defaultMaxMonth;
 }
 
+/**
+ * Gets the initial blank cells
+ *
+ * @param {Moment} momentDate A given moment date
+ * @returns {Array} Array of blanks
+ */
 function getInitialBlankCells(momentDate) {
   const firstWeekday = getWeekdayOfFirstOfMonth(momentDate);
 
-  if (firstWeekday === 0 || firstWeekday === 6) {
+  if (firstWeekday === '0' || firstWeekday === '6') {
```

Comment:
Changed to strings

Reply:
I see why the change to String but would it be better if it was a number? Reason being is that days of the week are typically numbers (0 - 6). Just a thought.

Reply:
This following would not be needed as well, 

let dayOfWeek = **Number**(getWeekdayOfFirstOfMonth(momentDate));